.buildkite/pipeline.yml

```diff
@@ -23,28 +62,7 @@ steps:
       - "be_cmd buck2-kill -- buck2 kill"
       - "be_cmd buck2-build -- buck2 build --console simple //:chromium"
       - "popd"
-    env:
-      GOMA_SERVER_HOST: simpsonite.goma.engflow.com
-      GOMACTL_USE_PROXY: false
-      RECORD_REPLAY_BACKEND_DIR: /home/ubuntu/chromium/backend
-    agents:
-      - "runtimeType=chromiumbuild"
-      - "os=linux"
-      - "queue=runtime"
-    artifact_paths:
-      - "build_id/linux/x86_64/**"
-  - label: "Build chromium - Mac (x86)"
-    key: "build-chromium-mac-x86_64"
-    timeout_in_minutes: 60
-    plugins:
-      - thedyrt/skip-checkout#v0.1.1:
-          cd: /Users/administrator/chromium/src
-      - seek-oss/aws-sm#v2.3.1:
-          region: us-east-2
-          env:
-            BUILDEVENT_APIKEY: honeycomb-api-key
-            BUILDEVENT_BUILDKITE_API_TOKEN: buildkite-api-token-honeycomb-build-events
-      - replayio/buildevents#adb8a05: ~
+  - &buckle_build
```

Comment:
q: is there a reason not to use buckle everywhere? that would make it so we can reuse the same build command sequence across linux/macos.

Reply:
the buck2/buckle build anchors are identical except for two lines.  in fact except for two words.  This is another place where yaml list merging (and string concatenation) would help a bunch.  I didn't really want to create anchors for each shared command here, like I did up in `plugins`.

Reply:
Eventually we'll be able to use buckle everywhere, but there's currently a bug in the Windows version https://github.com/benbrittain/buckle/pull/21